resources/views/courses/show.blade.php

```diff
@@ -78,81 +79,72 @@
                     <h6><span class="font-weight-bold">@lang("messages.course.price"):</span>
                         {{ $data["course"]->getPrice() }}
                     </h6>
-
                     @if (!$data["course"]->lessons->isEmpty())
-                    <h6><span class="font-weight-bold">@lang("messages.course.lessons"):</span></h6>
-                    <div class="list-group">
-                        @foreach ($data["course"]->lessons as $lesson)
-                        <a href="{{ route('lesson.show', $lesson->getId()) }}"
-                            class="list-group-item list-group-item-action">
-                            {{ $lesson->getTitle() }}
-                        </a>
-                        @endforeach
-                    </div>
-                    @else
-                    <h6><span class="font-weight-bold">@lang("messages.course.noLessons")</span></h6>
-                    @endif
-
-                </div>
-            </div>
-            <div class="card">
-                <div class="card-header">@lang("messages.course.reviews")</div>
-                <div class="card-body">
-                    @if (!$data["course"]->reviews->isEmpty())
-                    <ul class="list-group">
-                        @foreach ($data["course"]->reviews as $review)
-                        <li class="list-group-item">
-                            <h6><span class="font-weight-bold">@lang("messages.course.review.user"):</span>
-                                {{ $review->user->getName() }}</h6>
-                            <h6><span class="font-weight-bold">@lang("messages.course.review.rating"):</span>
-                                {{ $review->getRating() }}</h6>
-                            <h6><span class="font-weight-bold">@lang("messages.course.review.comment"):</span>
-                                {{ $review->getComment() }}</h6>
-                        </li>
-                        @endforeach
-                    </ul>
-                    @else
-                    <h6><span class="font-weight-bold">@lang("messages.course.hasNoReviews")</span></h6>
-                    @endif
+                    @include('lesson.list')
                 </div>
+                @else
+                <h6><span class="font-weight-bold">@lang("messages.course.noLessons")</span></h6>
+                @endif
             </div>
         </div>
-        @if ( $data['isEnrolled'] )
```

Comment:
This piece of code conflicts with the current state of the master branch, I believe the changes were introduced in #13.